Added countries command

benolot opened the pull request.

benolot committed:
Added Country Information command

benolot committed:
Added Country Information command

benolot committed:
Removed double https

benolot committed:
Removed double https

benolot committed:
Added country flags to country command

benolot committed:
Added country flags to country command

Khaaz committed:
Added Chuck Norris command (#13)

briantanner commented on src/commands/index.js:
```diff
@@ -9,4 +9,5 @@ exports.Rps = require('./Rps');
-exports.Pokemon = require('./Pokemon');
\ No newline at end of file
+exports.Pokemon = require('./Pokemon');
+exports.Country = require('.Country');
```
This is bugged, should be `./Country`

briantanner commented on src/commands/Country.js:
```diff
@@ -0,0 +1,95 @@
+        this.expectedArgs = 1;
+    }
+
+    async execute({ message }) {
```
`args` is missing here

benolot committed:
Gramatical changes on the embed

benolot committed:
Gramatical changes on the embed

benolot committed:
Fixed not exporting bug

salesawy1 committed:
Add space command (#16)

Khaaz committed:
Added Dad Joke command (#15)

* Added Dad Joke command

* fix in error message (bad english)

* Added Joke command.

briantanner committed:
minor cleanup, change cooldowns, disable movie command until api key is obtained

briantanner committed:
fix bug in pokemon introduced in cleanup

Khaaz committed:
Error message in define + roll command improvement (#17)

* Changed error message in define if it can't find the definition

* unable to find definition

* changed the error message again. This one should be more appropriate

* Improve roll command to support size. Also fix a bug where roll was sending an empty input if there re was an arg but not a number

* fixed indentation with tab (no space)

* Update Roll.js

fixed bad spacing

* changed how d00 was working to the real function of the dice

* Update Roll.js

Typo (d00 checker)

* another typo.. last commit!

* Update Roll.js

Fix last commit that was borked for some reason.

EleosOS committed:
Add adorable command (#19)

* Add adorable command

* Still adding it

* Update index.js

briantanner committed:
formatting

briantanner committed:
join args properly

benolot committed:
Bug fixes + country name search

benolot committed:
Bug fixes + country name search

benolot committed:
Merge remote-tracking branch 'origin/develop' into develop

benolot committed:
Merge remote-tracking branch 'origin/develop' into develop
